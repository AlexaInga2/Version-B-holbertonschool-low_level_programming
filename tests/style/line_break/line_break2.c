#include <stdio.h>

/**
 * main - Entry point
 *
 * Return: Always 0 (Success)
 */
int main(void)
{
	printf("hello"); /* a =
	12; return a; */
	printf("hello"); /* a = 12; return a;*/
	printf("hello"); a = 12; return a;
}
