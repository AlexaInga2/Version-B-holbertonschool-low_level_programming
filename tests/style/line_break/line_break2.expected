tests/style/line_break/line_break2.c:11: WARNING: Block comments use * on subsequent lines
tests/style/line_break/line_break2.c:11: WARNING: Block comments use a trailing */ on a separate line
tests/style/line_break/line_break2.c:13: WARNING: multiple instructions on a single line
total: 0 errors, 3 warnings, 14 lines checked
